e2e/testcases/apiservice.bats

```diff
@@ -38,18 +56,27 @@ test_teardown() {
   wait::for -t 240 -- nomos::repo_synced
 }
 
-@test "importer and syncer resilient to bad APIService" {
+@test "importer and syncer resilient to flaky APIService" {
+  # Create an APIService without backend so that Config Sync DiscoveryClient will
+  # get service unavailable errors when trying to parse resource
   debug::log "Adding bad APIService"
   kubectl apply -f "${YAML_DIR}/apiservice.yaml"
 
-  debug::log "Creating commit with Deployment"
+  debug::log "Creating a commit with some test resources"
   git::add \
     "${YAML_DIR}/namespace-resilient.yaml" \
     acme/namespaces/resilient/namespace-resilient.yaml
   git::commit
 
+  debug::log "Waiting for APIServerError to be present in Rootsync status"
```

Comment:
nit: The `nomos::source_error_code` function works for both mono-repo and multi-repo. Suggest removing `Rootsync` in the comment as that is specific to the multi-repo mode.

Reply:
Done